src/ontology/mondo-edit.obo

```diff
@@ -432559,9 +432560,9 @@ xref: ICD9:277.1 {source="DOID:13268"}
 xref: MedDRA:10036181 {source="Orphanet:738", source="Orphanet:738/e"}
 xref: MedDRA:10061356 {source="Orphanet:738", source="Orphanet:738/e"}
 xref: MEDGEN:698423 {source="MONDO:equivalentTo", source="MONDO:MEDGEN"}
-xref: MESH:D011164 {source="MONDO:equivalentTo", source="Orphanet:738", source="DOID:13268", source="Orphanet:738/e"}
+xref: MESH:D011164 {source="MONDO:relatedTo", source="Orphanet:738", source="DOID:13268", source="Orphanet:738/e"}
 xref: NANDO:2200610 {source="MONDO:NANDO", source="https://orcid.org/0000-0003-0011-764X", source="https://orcid.org/0000-0002-0170-9172"}
-xref: NCIT:C97096 {source="MONDO:equivalentTo", source="DOID:13268"}
+xref: NCIT:C97096 {source="MONDO:relatedTo", source="DOID:13268"}
```

Comment:
This x-ref refers to porphyria. If you want to add it here, the correct source would be MONDO:mondoIsNarrowerThanSource (and not MONDO:relatedTo).

Reply:
fixed